js/citipix.js

```diff
@@ -0,0 +1,67 @@
+$(function(){
+
+  // 1st I need to declare a variable so that whatever city is put in, i have storage for it
+
+   var city;
+
+   function getCity(){
+    var city = $("#city-type").val();
```

Comment:
@agreen2017 consider converting the input to the same case that you are trying to validate to